account_payment_order/tests/test_account_payment.py

```diff
@@ -16,7 +16,7 @@ def setUpClass(cls):
 
         # INSTANCES
         # Payment methods
-        cls.inbound_payment_method_01, cls.inbound_payment_method_02 = cls.payment_method_model.search(
+        cls.inbound_pm_01, cls.inbound_pm_02 = cls.payment_method_model.search(
```

Comment:
Why changing the variable names that doesn't add any added value?

Reply:
Because of this error:
hookid: flake8
account_payment_order/tests/test_account_payment.py:19:104: B950 line too long (103 > 80 characters)
The auto format changes it always back to the original, that's why I renamed them

Reply:
Well, it's easy to bypass the problem putting a parenthesis instead of renaming the variables across the whole file:
```suggestion
        cls.inbound_pm_01, cls.inbound_pm_02 = (
            cls.payment_method_model.search(...))
```

Reply:
Solved

Reply:
Solved!